src/java/fr/paris/lutece/plugins/workflowcore/service/provider/AbstractProviderManager.java

```diff
@@ -33,18 +33,11 @@
  */
 package fr.paris.lutece.plugins.workflowcore.service.provider;
 
-import java.util.Collection;
-
-import jakarta.servlet.http.HttpServletRequest;
-
-import fr.paris.lutece.plugins.workflowcore.business.resource.ResourceHistory;
-import fr.paris.lutece.plugins.workflowcore.service.task.ITask;
-
 /**
  * This class represents a manager of providers
  *
  */
-public abstract class AbstractProviderManager
+public abstract class AbstractProviderManager implements IProviderManager
```

Comment:
Add the @Deprecated annotation with a Javadoc comment to indicate that it should implement the interface directly